tests/runtests.py

```diff
@@ -21,7 +21,7 @@ def load_suite_tests(only=None):
             basename, ext = os.path.splitext(f)
             if (ext == '.py') and (not only_module or (only_module == basename)):
                 modname = "%s.%s" % ('.'.join(dirpath.split('/')), basename)
-                package = __import__(modname, globals(), locals(), [], -1)
+                package = __import__(modname, globals(), locals(), [], 0)
                 mod = sys.modules[modname]
```

Comment:
Ah ok.
I can't remember if I set to -1 intentionally to avoid the tests to run against the installed lib (instead of the dev. sources)...
But this should be handled with the `sys.path.insert()` at the top of the file.
I'll check that.